sdk/ai/azure-ai-resources/setup.py

```diff
@@ -70,6 +70,7 @@
         "mlflow-skinny<3",
         "azureml-telemetry~=1.0,>=1.51.0",
         "azure-mgmt-resource<23.0.0,>=22.0.0",
+        "tqdm",
```

Comment:
Where is this used?

Reply:
https://github.com/Azure/azure-sdk-for-python/blob/ad5ddf4e71341e51ce09b36342bf52fe73a555b3/sdk/ai/azure-ai-resources/azure/ai/resources/_index/_embeddings/openai.py#L204

Reply:
I see. It looks like an optional dependency. Let's keep it out of the install_requires list to minimize the dependencies. The AI SDK now suffers the problem of too many dependencies and it takes up to 20 mins to install both the ai-resources and ai-generative packages. I want to be extra careful about adding new dependencies.

Reply:
In fact, I added it there because I was trying to fix followng build error -